### ISSUE TITLE
Rails configuration not loaded on rake tasks

### ISSUE THREAD

**Issue by StefSchenkelaars** [ElMassimo/vite_ruby#193]
[troubleshooting section]: https://vite-ruby.netlify.app/guide/troubleshooting.html

- [x] I have tried upgrading by running `bundle update vite_ruby`.
- [x] I have read the __[troubleshooting section]__ before opening an issue.

### Description 📖
When you run a vite rake task, the [rails configuration overrides](https://github.com/ElMassimo/vite_ruby/blob/main/vite_rails/lib/vite_rails/config.rb#L8) are not properly loaded. 

### Reproduction 🐞
Consider the case where the rails assets host is set: `Rails.application.config.asset_host = "http://assets.lvh.me:3000"`
And a `foobar` rake task is defined:

```ruby
task foobar: :environment do
  puts "-- ViteRuby.config.asset_host: #{ViteRuby.config.asset_host.inspect}"
end
```

If you run `rake foobar` you get the expected outcome:
```
$ bundle exec rake foobar           
-- ViteRuby.config.asset_host: http://assets.lvh.me:3000
```

However, if you prefix the command with any vite rake task, the rails configuration is not properly loaded:
```
$ bundle exec rake vite:clean foobar
-- ViteRuby.config.asset_host: nil
```

<details>
<summary>Vite Ruby Info</summary>

_Run `bin/rake vite:info` and provide the output:_

```
bin/vite present?: true
vite_ruby: 3.0.9
vite_rails: 3.0.6
rails: 7.0.2.3
vite_plugin_legacy: 3.0.2
rack: 2.2.3
node: v16.13.1
npm: 8.1.2
yarn: 1.22.17
pnpm: 6.24.1
ruby: ruby 2.7.5p203 (2021-11-24 revision f69aeb8314) [x86_64-darwin20]

installed packages:
xxx
│ ├─┬ vite-plugin-style-import@1.4.1
│ │ └── vite@2.8.2 deduped
│ └── vite@2.8.2
├─┬ vite-plugin-ruby@3.0.8
│ └── vite@2.8.2 deduped
├─┬ vite-plugin-style-import@1.4.1
│ └── vite@2.8.2 deduped
└── vite@2.8.2
```
</details>

### Logs 📜

_If not providing a reproduction:_

<details>
<summary>Output</summary>

_Run `DEBUG=vite-plugin-ruby:* bin/vite dev` or `DEBUG=vite-plugin-ruby:* bin/vite build` and provide the output:_

```

```
</details>

### Screenshots 📷

_Provide console or browser screenshots of the problem_.

**Comment by ElMassimo** [comment]
Hi Stef, thanks for reporting!

I think a reasonable fix would be to have `vite_rails` [extend](https://github.com/ElMassimo/vite_ruby/blob/fix/strict-dev-proxy/vite_rails/lib/tasks/vite.rake#L4) the `vite:verify_install` rake task to depend on the `environment` rake task defined by Rails.

That would ensure the Rails application is loaded before Vite Ruby resolves the config.

PRs are welcome!

**Comment by ElMassimo** [comment]
Released a fix in [`vite_rails@3.0.7`](https://github.com/ElMassimo/vite_ruby/blob/main/vite_rails/CHANGELOG.md#307-2022-03-17).

**Comment by StefSchenkelaars** [comment]
@ElMassimo thanks again! I tried writing a test for it but that wasn't easy. 
But nice that the fix is out 😊
